docs/src/refman/graphics.txt

```diff
@@ -1144,30 +1198,6 @@ Return the multi-sampling samples used by this bitmap if it is used with
 
 Since: 5.1.14
 
-### API: al_set_bitmap_depth
```

Comment:
It's too bad this had to go.  I really liked the idea and simplicity of `al_set_bitmap_depth` and `al_set_bitmap_samples`. :blush: